### PR TITLE
Deprecated getLanguageName. Added getPluginName.

### PR DESCRIPTION
**Pull request by WaDelma**
Fixes issue #14

### CODE REVIEW

File: tmc-langs-framework/src/main/java/fi/helsinki/cs/tmc/langs/LanguagePlugin.java [hunk 1]
```diff
@@ -27,10 +27,18 @@
 public interface LanguagePlugin {
 
     /**
+     * Returns the name of the plug-in
+     *
+     * @return The name of the plug-in
```
Comment (jamox) [inline tmc-langs-framework/src/main/java/fi/helsinki/cs/tmc/langs/LanguagePlugin.java]: We can drop the @returns part of the javadoc

File: tmc-langs-framework/src/main/java/fi/helsinki/cs/tmc/langs/LanguagePlugin.java [hunk 2]
```diff
@@ -27,10 +27,18 @@
...
+     */
+    String getPluginName();
+
+    /**
      * Returns the name of the programming language supported by this plug-in.
      *
      * @return The name of the language supported by this plug-in.
```
Comment (jamox) [inline tmc-langs-framework/src/main/java/fi/helsinki/cs/tmc/langs/LanguagePlugin.java]: Let's drop it also from here.

**Comment by jamox** [comment]
LGTM, please fix the two comments

**Comment by WaDelma** [comment]
Is everything fine with this commit? It decreases code coverage, because it adds a method that replaces old one. If the tests were changed to use new one it still would do that.

File: tmc-langs-framework/src/main/java/fi/helsinki/cs/tmc/langs/LanguagePlugin.java [hunk 3]
```diff
@@ -27,10 +27,16 @@
 public interface LanguagePlugin {
 
     /**
+     * Returns the name of the plug-in.
+     *
```
Comment (jamox) [inline tmc-langs-framework/src/main/java/fi/helsinki/cs/tmc/langs/LanguagePlugin.java]: Rm blank line

File: tmc-langs-framework/src/main/java/fi/helsinki/cs/tmc/langs/LanguagePlugin.java [hunk 4]
```diff
@@ -27,10 +27,16 @@
...
+     */
+    String getPluginName();
+
+    /**
      * Returns the name of the programming language supported by this plug-in.
      *
```
Comment (jamox) [inline tmc-langs-framework/src/main/java/fi/helsinki/cs/tmc/langs/LanguagePlugin.java]: Rm blank line

**Comment by jamox** [comment]
Added one last comment.

We don't get notified when you push new changes to the pr...

**Comment by WaDelma** [comment]
Yeah I realised it from an irc conversation :smile:

**Pull request merged**